src/Particle/Lattice/ParticleBConds3DSoa.h

```diff
@@ -428,7 +428,7 @@ struct DTD_BConds<T, 3, PPPG + SOA_OFFSET>
 
       T rmin = delx * delx + dely * dely + delz * delz;
       int ic = 0;
-#pragma unroll(7)
```

Comment:
This seems to be a syntactic change. Does this really change anything?

Reply:
IMO, it's questinable at best to tell the compiler how much to unroll.
But that's that the code does. Not my call.
I've never seen the number in parentheses, tough, and hipcc complains about it.
I am leaning towards leaving it without parentheses.

Reply:
Sorry I didn't express my question clearly. I'm not asking about how good unroll works. I'm wondering if changing syntax really has impact.
I saw more compiler support with parentheses being added.
Clang clearly supports it
https://clang.llvm.org/docs/AttributeReference.html#pragma-unroll-pragma-nounroll

This file should not be compiled by hipcc. Do you have the warning at hand?

Reply:
Good point.
It's meant for the host compiler.
It's only a warning with hipcc, which I can easily suppress with a flag.
Let me undo this then.

Reply:
BTW, the warning is, e.g.:
/mnt/shared/container-share/qmcpack/src/Particle/Lattice/ParticleBConds3DSoa.h:431:16: warning: argument to '#pragma unroll' should not be in parentheses in CUDA C/C++ [-Wcuda-compat]
#pragma unroll(7)

Reply:
Likely you were using hipcc as your host compiler. We don't recommend this way.